Makefile

```diff
@@ -53,7 +53,7 @@ endif
 GOARCH?=$(shell $(GO_CMD) env GOARCH)
 
 # Local (alternative) GOBIN for auxiliary build tools
-GOBIN_ALT:=$(CURDIR)/.bin/
+GOBIN_ALT:=$(CURDIR)/.bin
```

Comment:
This change seems unrelated to the kustomize/kubectl deprecation of "bases". If this is a useful change I'd like to see it split out and described, thanks.

Reply:
Done.